### PR TITLE
Add react-router

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/dashboard/dashboard-manager.tsx [hunk 1]
```diff
@@ -60,26 +61,38 @@ const DashboardManager: FC = (): JSX.Element => {
...
-        <Dashboard
-          hidden={shouldShowDashboard}
-          rows={rows}
-          userWalletAddress={userWalletAddress}
-          onRowActionClick={onRowActionClick}
-        />
-      )}
+    <BrowserRouter>
+      <Routes>
+        <Route
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/dashboard/dashboard-manager.tsx:66]: can we somehow separate the routes from the rendered components?

Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/dashboard/dashboard-manager.tsx:66]: I can create a route component  that is responsible for rendering other components on that route.

**Review by danieledamiani — DISMISSED** [review]

File: packages/tempus-client_v2/src/components/dashboard/dashboard-manager.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/dashboard/dashboard-manager.tsx:66]: Should I make this fix in this PR, or do that in next PR?

**Comment by lazarmitic** [comment]
Implemented requested change.

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**